scripts/build/emmet.cmake

```diff
@@ -6,7 +6,7 @@ endmacro()
 set_cache_var(CELERITAS_USE_CUDA BOOL ON)
 set_cache_var(CELERITAS_USE_Geant4 BOOL ON)
 set_cache_var(CELERITAS_USE_HepMC3 BOOL ON)
-set_cache_var(CELERITAS_USE_MPI BOOL ON)
+set_cache_var(CELERITAS_USE_MPI BOOL OFF)
```

Comment:
Is this change of cmake default indeed to be part of this PR?

Reply:
Ah no, I can revert this.